packages/docs-site/src/library/pages/components/buttongroup.md

```diff
@@ -0,0 +1,303 @@
+---
+title: Button Group
+description: A collection of buttons commonly used to select an option
+header: true
+---
+
+import { Button,ButtonGroup,  Icons, Tab, TabSet } from '@royalnavy/react-component-library'
```

Comment:
Spacing could be tweaked here

Reply:
This file is still being worked on by Ed.

Reply:
Waiting for design documentation to be added